app/components/upload-app/via-link/index.hbs

```diff
@@ -86,6 +88,7 @@
         >
           <AkButton
             data-test-uploadAppViaLinkModal-confirmBtn
+            data-test-cy='uploadAppViaLinkModal-confirmBtn'
```

Comment:
Since this is a button, use the `findByRole` to find this button and use the label to specifically target it.